repository-data/application/src/main/resources/hcm-config/configuration/update/Migration20180502RevertExternalAttachmentsToEmbedded.groovy

```diff
@@ -0,0 +1,47 @@
+package org.hippoecm.frontend.plugins.cms.admin.updater
+
+import org.hippoecm.repository.util.JcrUtils
+import org.onehippo.repository.update.BaseNodeUpdateVisitor
+
+import javax.jcr.Node
+
+import static org.hippoecm.repository.HippoStdNodeType.NT_RELAXED
+
+/**
+ * Swap the pre uploaded attachment node to using the S3 resource instead of the embedded one
```

Comment:
This comment probably needs to be the other way around (presumably copied from original script)?